src/zrc/api/viewsets.py

```diff
@@ -39,36 +39,36 @@ class ZaakViewSet(GeoMixin,
                   mixins.ListModelMixin,
                   mixins.RetrieveModelMixin,
                   mixins.UpdateModelMixin,
+                  mixins.DestroyModelMixin,
                   viewsets.GenericViewSet):
     """
     Opvragen en bewerken van ZAAKen.
 
-    Een zaak mag (in principe) niet meer gewijzigd worden als de `archiefstatus`
-    een andere status heeft dan "nog_te_archiveren". Voor praktische redenen
-    is er geen harde validatie regel aan de provider kant.
+    Een zaak mag (in principe) niet meer gewijzigd worden als de `archiefstatus` een andere status heeft dan
+    "nog_te_archiveren". Voor praktische redenen is er geen harde validatie regel aan de provider kant.
 
     create:
     Maak een ZAAK aan.
 
-    Indien geen identificatie gegeven is, dan wordt deze automatisch
-    gegenereerd. De identificatie moet uniek zijn binnen de bronorganisatie.
+    Indien geen identificatie gegeven is, dan wordt deze automatisch gegenereerd. De identificatie moet uniek zijn
```

Comment:
docstrings follow a convention of shorter-than-usual length, see https://www.python.org/dev/peps/pep-0008/#id19

I prefer to stick to < 80 charachters in docstrings, otherwise my editor popups with docstrings of imported stuff requires sideways scrolling which is pretty annoying